sysid/drivetrain/config.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
This file is for a particular robot, right? Should we save with a more descriptive name?